src/Compilers/Test/Core/AssemblyLoadTestFixture.cs

```diff
@@ -110,11 +110,65 @@ public sealed class AssemblyLoadTestFixture : MarshalByRefObject, IDisposable
 
         public string AnalyzerWithLaterFakeCompilerDependency { get; }
 
+        /// <summary>
+        /// An analyzer that can be used to load a resource
+        /// </summary>
+        public string AnalyzerWithLoc { get; }
+
+        /// <summary>
+        /// An analyzer that can be used to load a resource
+        /// </summary>
+        public string AnalyzerWithLocResourceEnGB { get; }
+
         public AssemblyLoadTestFixture()
         {
             _temp = new TempRoot();
             _directory = _temp.CreateDirectory();
 
+            const string AnalyzerWithLocSource = """
+using System;
+using System.Resources;
+using System.Reflection;
+using System.Threading;
+using System.Globalization;
+
+[assembly: System.Reflection.AssemblyTitle("AnalyzerWithLoc")]
+[assembly: System.Reflection.AssemblyVersion("1.0.0.0")]
+
+namespace AnalyzerWithLoc
+{
+    public static class Util
+    {
+        public static void Exec(string culture)
+        {
+            try
+            {
+                var ci = new CultureInfo("en-GB");
+                var rm = new ResourceManager("rmc", typeof(Util).Assembly);
+                _ = rm.GetString("hello", ci);
+            }
+            catch (Exception ex)
```

Comment:
It's needed because the resource doesn't actually exist. This code is just getting the runtime to probe for the satellite assemblies to prove that we do the loading correct. So the satellite load will succeed then it will throw when it can't find the resource. 

Getting a real resource was going to involve checking in a couple of binary resources that I couldn't think of an easy way to maintain. As such decided to just focus on testing the load portion here.